AllEvents.TicketManagement/src/Core/AllEvents.TicketManagement.Application/Contracts/IEventRepository.cs

```diff
@@ -4,9 +4,16 @@ namespace AllEvents.TicketManagement.Application.Contracts
 {
     public interface IEventRepository
     {
-        Task<List<Event>> GetPagedEventsAsync(int pageIndex, int pageSize, bool includeDeleted = false);
+        Task<List<Event>> GetFilteredPagedEventsAsync(
```

Comment:
@KKDinev20 no need to add this one to repository 
use the IEventQuery interface directly in the ...QueryHandler.cs class 
so remove those one